bodhi/server/util.py

```diff
@@ -312,7 +312,7 @@ def sanity_check_repodata(myurl, repo_type):
 
         if repo_type in ('yum', 'source'):
             tests.append((['list', 'available'], 'testrepo'))
-        elif repo_type == 'module':
+        else:  # repo_type == 'module', verified above
```

Comment:
If there will be new `repo_type` in future (for example 'flatpak') we could easily forget about this change.

Reply:
The existing code would just silently do nothing in this case :wink:. I'll add asserts to guard against this.

Reply:
Raise an exception if the repo_type isn't supported?

Reply:
Yup, just through using asserts which don't (shouldn't) affect coverage.